ast/policy.go

```diff
@@ -835,12 +847,17 @@ func (rule *Rule) elseString() string {
 		buf = append(buf, value.String())
 	}
 
+	switch opts.regoVersion {
+	case RegoV1, RegoV0CompatV1:
+		buf = append(buf, "if")
+	}
```

Comment:
Just from looking at the code, I get the hunch that there's a `" "` missing between `"if"` and `"{"`?

Reply:
There is a `strings.Join(buf, " ")` just a few lines down.